### ISSUE TITLE
Problem plotting butterfly

### ISSUE THREAD

**Issue by rlopezcoto** [gammapy/gammapy#892]
Hi,

I get a problem trying to plot a butterfly from the Fermi catalogs with the latest gammapy version (0.6.dev4171). If I run:

examples/example_fermi_catalogs.py

I get:
```
python example_fermi_catalogs.py 
Traceback (most recent call last):
  File "example_fermi_catalogs.py", line 31, in <module>
    butterfly_3fg = src_3fgl.spectrum.butterfly(energy)
  File "/Users/rubenlopez/anaconda/lib/python2.7/site-packages/gammapy-0.6.dev4171-py2.7-macosx-10.6-x86_64.egg/gammapy/spectrum/results.py", line 281, in butterfly
    flux, flux_err = self.model.evaluate_error(energy)
  File "/Users/rubenlopez/anaconda/lib/python2.7/site-packages/gammapy-0.6.dev4171-py2.7-macosx-10.6-x86_64.egg/gammapy/spectrum/models.py", line 77, in evaluate_error
    upars = self.parameters._ufloats
  File "/Users/rubenlopez/anaconda/lib/python2.7/site-packages/gammapy-0.6.dev4171-py2.7-macosx-10.6-x86_64.egg/gammapy/utils/modeling.py", line 211, in _ufloats
    raise ValueError('Covariance matrix not set.')
```

**Comment by cdeil** [comment]
This week we introduced `gammapy.utils.modeling` and started using it also for spectral models.
The transition isn't finished, there's #889 still open and further changes might be needed.

Thanks for reporting this issue, we'll put another comment here and close it when it's fixed (within the next few days).

**Comment by cdeil** [comment]
This issue was fixed today by @adonath , probably via #889.

@rlopezcoto - Could you please update to latest Gammapy `master` and try to run `python examples/example_fermi_catalogs.py` again?

Works for me now:
![figure_1](https://cloud.githubusercontent.com/assets/852409/23139973/6421005e-f7af-11e6-999d-af69e2e9b404.png)

**Comment by rlopezcoto** [comment]
It works for me too, thanks!
